src/nix/describe.cc

```diff
@@ -0,0 +1,85 @@
+#include "command.hh"
+#include "eval-cache.hh"
+#include "shared.hh"
+#include "markdown.hh"
+
+#include <regex>
+
+using namespace nix;
+
+struct CmdDescribe : InstallableCommand
+{
+    std::optional<std::string> filter;
+
+    CmdDescribe()
+    {
+        addFlag({
+            .longName = "filter",
+            .description = "Only show options that match this regular expression.",
+            .labels = {"regex"},
+            .handler = {&filter},
+        });
+    }
+
+    std::string description() override
+    {
+        return "show information about a configurable derivation";
+    }
+
+    Category category() override { return catSecondary; }
+
+    // FIXME: add help
+
+    void run(ref<Store> store) override
+    {
+        std::optional<std::regex> filterRegex;
+        if (filter)
+            filterRegex = std::regex(*filter, std::regex::extended | std::regex::icase);
+
+        auto state = getEvalState();
+
+        auto cursor = installable->getCursor(*state);
+
+        auto type = cursor->getAttr(state->sType)->getString();
+
+        if (type != "configurable")
```

Comment:
Maybe it's overloading the concept, but it seems like you might expect `nix describe` to also work on flakes.